test/test_biot_savart.py

```diff
@@ -6,10 +6,12 @@
 
 from shimmingtoolbox.coils.biot_savart import biot_savart
 
+H_GYRO_R = 42.577478518e+6 # Hz/T
```

Comment:
I would rename and update the value to be the same as the other file in this PR. This value has more decimal so I would update the other value.